armi/reactor/converters/uniformMesh.py

```diff
@@ -537,6 +540,20 @@ def setAssemblyStateFromOverlaps(
                 destBlock, blockParamNames, cachedParams
             )
 
+            # If requested, the reaction rates will be calculated based on the
+            # mapped neutron flux and the XS library.
+            if calcReactionRates:
+                core = sourceAssembly.getAncestor(lambda c: isinstance(c, Core))
+                if core is not None:
+                    UniformMeshGeometryConverter._calculateReactionRates(
+                        lib=core.lib, keff=core.p.keff, assem=destinationAssembly
+                    )
+                else:
+                    runLog.warning(
+                        f"Reaction rates requested for {destinationAssembly}, but no core object exists. This calculation "
+                        "will be skipped."
```

Comment:
Will we get a zillion of these warnings? If so, do we want to add a `single = True` ?

Reply:
We can add single = True for sure!